test/__tests__/data/criteo_audience_router_rETL_input.json

```diff
@@ -0,0 +1,63 @@
+[
+  {
+    "destination": {
+      "Config": {
+        "clientId": "abcdef8-f49-4cd6-b4c5-958b3d66d431",
+        "clientSecret": "sjhdkhfrz6yc9LrRRIPimE9h53jADLccXTykHCcA6eEoFR4rXQg",
+        "audienceId": "34893",
```

Comment:
Can you add one more test case where `audienceId` is null or not available as that's the current code change ?

Reply:
added.